subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -493,10 +493,20 @@ config BT_CTLR_ADV_EXT_PDU_EXTRA_DATA_MEMORY
 	  that is part of PDU data.
 
 config BT_CTRL_ADV_ADI_IN_SCAN_RSP
+	bool "Include ADI in AUX_SCAN_RSP PDU [DEPRECATED]"
+	depends on BT_BROADCASTER && BT_CTLR_ADV_EXT
+	select BT_CTLR_ADV_ADI_IN_SCAN_RSP
+	select DEPRECATED
```

Comment:
A bit of a slippery slope effect here, but now that you did the formal deprecation it means that this should also be mentioned in `doc/releases/migration-guide-4.1.rst` (or maybe it was the release notes - please double check where deprecations are normally listed)